Content.Server/Procedural/DungeonJob.PostGen.cs

```diff
@@ -191,7 +192,8 @@ private async Task PostGen(BoundaryWallPostGen gen, Dungeon dungeon, EntityUid g
             if (dungeon.Entrances.Contains(neighbor))
                 continue;
 
-            if (!_anchorable.TileFree(grid, neighbor, DungeonSystem.CollisionLayer, DungeonSystem.CollisionMask))
+            if (!_anchorable.TileFree(new Entity<MapGridComponent>(gridUid, grid), neighbor, DungeonSystem.CollisionLayer,
```

Comment:
Use tuple syntax instead of `new Entity`.
For example `new Entity<MapGridComponent>(gridUid, grid)` should just be `(gridUid, grid)`.

(Do this in **all places** where you used `new Entity`, not just this one)